client/rest-high-level/src/test/java/org/elasticsearch/client/ml/GetBucketsRequestTests.java

```diff
@@ -18,47 +18,36 @@
  */
 package org.elasticsearch.client.ml;
 
-import org.elasticsearch.client.ml.job.util.PageParams;
+import org.elasticsearch.common.unit.TimeValue;
 import org.elasticsearch.common.xcontent.XContentParser;
 import org.elasticsearch.test.AbstractXContentTestCase;
 
 import java.io.IOException;
 
-public class GetBucketsRequestTests extends AbstractXContentTestCase<GetBucketsRequest> {
+public class GetBucketsRequestTests extends AbstractXContentTestCase<GetOverallBucketsRequest> {
```

Comment:
It looks like you duplicated this file then edited the original rather than the duplicate.  The newly added `GetOverallBucketsRequestTests.java` contains the test for `GetBucketsRequest` and the pre-existing `GetBucketsRequestTests.java` contains the test for `GetOverallBucketsRequest`.

Reply:
Good catch!